elliott/tests/test_verify_kernel_tag_cli.py

```
@@ -0,0 +1,471 @@
+        }
+        mock_get.return_value = mock_resp
+        result = get_kernel_rpms_from_rhcos("rhcos-x86_64-418.92.1-1", {"kernel"})
+        self.assertEqual(result, ["kernel-5.14.0-1.el9"])
+        mock_resp.raise_for_status.assert_called_once()
+
+    @patch("elliottlib.cli.verify_kernel_tag_cli.requests.get")
+    def test_http_error(self, mock_get):
+        mock_resp = MagicMock()
+        mock_resp.raise_for_status.side_effect = Exception("404 Not Found")
+        mock_get.return_value = mock_resp
+        with self.assertRaises(Exception, msg="404 Not Found"):
```

Ruff (0.16.1)

[warning] Do not assert blind exception: `Exception`

(B017)